charts/pipeline/values.yaml

```diff
@@ -190,12 +190,15 @@ configuration:
   #     # Enable create & update of EKS addons like coredns
   #     enableAddons: false
 
-  #   pke:
-  #     amazon:
+    pke:
+      amazon:
   #       globalRegion: us-east-1
   #       defaultImages: {}
   #       defaultNetworkProvider: "cilium"
   #       defaultNodeVolumeSize: 0 # GiB, 0/fallback: max(50, AMISize)
+        enabled: true
```

Comment:
**Recommendation**: let's move this enabled attribute to the top of the node.

Same in config/config.yaml.dist and internal/cmd/config.go

Reply:
Valid remark, done!